File: iniciante/python/1186-abaixo-da-diagonal-secundaria.py
def abaixo_da_diagonal_secundaria():
    operacao = input()
    matriz = list()
    linha = list()
    total = 0
    numero_de_celulas = 0
    aux = 11
    for i in range(12):
        for j in range(12):
            linha.append(float(input()))
        matriz.append(linha)
        linha = list()
    for k in range(12):
        for t in range(12):
            if t > aux:
                total += matriz[k][t]
                numero_de_celulas += 1
        aux -= 1
    if operacao == 'S':
        print(f'{total:.1f}')
    elif operacao == 'M':
        media = total / numero_de_celulas
        print(f'{media:.1f}')


abaixo_da_diagonal_secundaria()

File: iniciante/c/1186-abaixo-da-diagonal-secundaria.c
#include <stdio.h>

int main(){
    double matriz[12][12], soma = 0.0, media, count = 0.0; 
    int i, j, inicial = 12;
    char operacao;
    scanf("%c", &operacao);
    for(i = 0; i < 12; i++){
        for(j = 0; j < 12; j++){
            scanf("%lf", &matriz[i][j]);
        }
    }
    for(i = 0; i < 12; i++){
        for(j = inicial; j < 12; j++){
            soma += matriz[i][j];
            count = count + 1.0;
        }
        inicial--;
    }
    if (operacao == 'S'){
        printf("%.1lf\n", soma);
    }
    else{
        media = soma / count;
        printf("%.1lf\n", media);
    }
    return 0;
}
